selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java

```diff
@@ -215,7 +216,19 @@ public void loadMoreActivities() {
     evt.waitForAndGetElement(ELEMENT_PLF_HOMEPAGE_LOAD_MORE_BUTTON, testBase.getDefaultTimeout(), 1);
     evt.click(ELEMENT_PLF_HOMEPAGE_LOAD_MORE_BUTTON);
   }
-
+public void refreshUntil(String text, Condition condition,long time){
```

Comment:
Indentation is not good.
Also, add a blank line before and after the method.

Reply:
code formatted